detection-rules/attachment_macro_dll_loader.yml

```diff
@@ -6,33 +6,35 @@ references:
 type: "rule"
 severity: "high"
 source: |
-  type.inbound 
-  and any(attachments, .file_extension == "zip"
-         and (any(file.explode(.),
-              .scan.zip.encrypted == false
-              // zip contains a dll file
-              and any(.scan.zip.all_paths, strings.icontains(., "dll")))
-          and any(file.explode(.),
-              // macro references a dll file
-              any(.flavors.yara, strings.like(., "vb_file"))
-              and any(.scan.strings.strings, strings.icontains(., "dll"))
-             )
-         )
-         or any(file.explode(.), // fallback for encrypted zips
-             .scan.zip.encrypted == true
-             and any(.scan.zip.all_paths, any($file_extensions_macros, strings.icontains(.., .)))
-             // zip contains a dll file
-             and any(.scan.zip.all_paths, strings.icontains(., "dll"))
-         )
-  or any(attachments, .file_extension in~ $file_extensions_common_archives
```

Comment:
This `any(attachments, ...)` was actually nested inside another `any(attachments, ...)`!